ui-tests/jupyter-lite.json

```diff
@@ -2,6 +2,7 @@
   "jupyter-lite-schema-version": 0,
   "jupyter-config-data": {
     "appName": "JupyterLite UI Tests",
+    "defaultKernelName": "xlua",
```

Comment:
@martinRenou @DerThorsten is there anything in `xeus-lua` or `xeus-python` that could explain the sudden change noticed in https://github.com/jupyterlite/xeus/issues/83#issue-2212620796 ?

Reply:
It could have been the new JupyterLite `0.3.0`, but it looks pinning down in https://github.com/jupyterlite/xeus/pull/84#issuecomment-2024881646 produces the same outcome.